web/components/Canvas/MainMenu/MainMenu.jsx

```diff
@@ -25,35 +35,35 @@ function MainMenu({ setGameId }) {
   };
 
   return (
-    <div className={styles.MainMenu}>
-      <div className={styles.buttons}>
-        <div className={styles.newGame}>
+    <Card className={styles.MainMenu}>
+      <CardContent>
+        <Typography className={styles.menuTitle}>
+          Welcome to Clueless
+        </Typography>
+        <Container className={styles.userId}>
+          <TextField
+            onChange={(e) => setClientIdInput(e.target.value)}
+            label="Display Name"
+            variant="standard"
+            helperText="Please enter your display name."
+            fullWidth
+            required

```

Comment:
required not actually hooked up to form validation. need to rework slightly to make this into a form submit and run validation to prevent submittion when name is empty

Reply:
Yup, I added the `required `prop to display the visual cue that the field is required. But you are totally right, we are going to need form validation here and also on the join game input field. I'll go back and add it for both

Reply:
Added the form validation. PTAL.